python/ql/lib/semmle/python/frameworks/FlaskAdmin.qll

```diff
@@ -0,0 +1,79 @@
+/**
+ * Provides classes modeling security-relevant aspects of the `Flask-Admin` PyPI package
+ * (imported as `flask_admin`).
+ *
+ * See
+ * - https://flask-admin.readthedocs.io/en/latest/
+ * - https://pypi.org/project/Flask-Admin/
+ */
+
+private import python
+private import semmle.python.dataflow.new.DataFlow
+private import semmle.python.dataflow.new.RemoteFlowSources
+private import semmle.python.dataflow.new.TaintTracking
+private import semmle.python.Concepts
+private import semmle.python.frameworks.Flask
+private import semmle.python.ApiGraphs
+
+/**
+ * Provides models for the `Flask-Admin` PyPI package (imported as `flask_admin`).
+ *
+ * See
+ * - https://flask-admin.readthedocs.io/en/latest/
+ * - https://pypi.org/project/Flask-Admin/
+ */
+private module FlaskAdmin {
+  /**
+   * A call to `flask_admin.expose`, which should be used as a decorator to make the
```

Comment:
The use of "should be" makes me think it's saying something about the API _we're_ implementing here, but rather it's about how that function is intended to be used in the Python code itself. Since this is a `private` module, I guess it doesn't matter too much either way.

Reply:
Would this make it better?

```suggestion
   * A call to `flask_admin.expose`, which is used as a decorator to make the
```

Reply:
Yeah, that sounds less confusing to me.